quickstart-mongodbatlas/README.md

```diff
@@ -0,0 +1,29 @@
+quickstart-mongodbatlas
+=============
+## MongoDB Atlas on the AWS Cloud
+
+Start here: [beta-launch.md](beta-launch.md)
+
+This Quick Start sets up a flexible, scalable AWS environment for MongoDB Atlas through AWS CloudFormation resources.
+
+MongoDB Atlas is an open source, NoSQL database that provides support for JSON-styled, document-oriented storage systems. 
+Its flexible data model enables you to store data of any structure, and it provides full index support, sharding, and replication.
+
+The Quick Start offers two deployment options:
+
+- Deploying MongoDB Atlas into a new virtual private cloud (VPC) on AWS
+- Deploying MongoDB Atlas into an existing VPC on AWS
+
+You can also use the (sample)[sample] AWS CloudFormation templates as a starting point for your own implementation.
+
+:construction: 
+
+![Quick Start architecture for MongoDB on AWS](atlas-cfn-dataplatform.png)
+
+:construction: 
+
+For architectural details, best practices, step-by-step instructions, and customization options, see the 
+[deployment guide](atlas-cfn-dataplatform.png).
+
+To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
+If you'd like to submit code for this Quick Start, please review the [AWS Quick Start Contributor's Kit](https://aws-quickstart.github.io/). 
```

Comment:
Do you want to point to our contributor file?

Reply:
until we figure out the flow to publish the quickstart to the aws repositories, I think just remove this.
then we can add back one consistent story about how to contribute.